### ISSUE TITLE
Extend hedwig AWS Jenkins/CodeBuild project to analyze & post technical debt

### ISSUE THREAD

**Issue by speedcm** [ctoec/ecis-experimental#27]
Whenever CodeBuild performs a build, the source should be statically analyzed to determine the technical debt & the results should be posted.  

SonarCloud was selected as the tool to manage technical debt. However, AWS CodeBuild does not have a built-in integration with SonarCloud. 

To achieve SonarCloud integration with AWS CodeBuild, the following must be done:

1. Sonar account must be setup for company on sonarqube.org

1. Sonar project must be setup to support hedwig on sonarqube.org

1. Unique GUID must be embedded in the hedwig.csproj file

1. Sonar connection must be initialized in the CodeBuild buildspec.yml file

1. Sonar scanner must be executed in the CodeBuild buildspec.yml file

1. Sonar results must be posted via the CodeBuild buildspec.yml file

When this issue is complete:

- all technical debt will be stored at sonarqube.org
- the world will be able to see the technical debt reports 
- the world will be able to drill down to source 
- the world will be able to see source associated with debt issues

Notes:

- Posting to sonarqube.org requires a unique GUID in the dotnetcore project 
- A GUID may not be embeded into a dotnetcore project file upon its creation
- A GUID can be obtained from https://www.guidgenerator.com/online-guid-generator.aspx

This issue is dependent upon issue #24

**Comment by speedcm** [comment]
A DevOps meeting was held on 09/30/2019 with the management team to discuss the implementation of Jenkins to support AWS DevOps.  The Jenkins effort has been put on hold in favor of extending Azure DevOps 2019 to support AWS RDS and AWS Beanstalk deployments. As a result, this issue is cancelled.
